bindings/java/gateway-java-binding/src/main/java/com/microsoft/azure/gateway/core/IGatewayModule.java

```diff
@@ -7,8 +7,7 @@
 public interface IGatewayModule {
 
     /**
-     * The create method is called by the subclass constructor when the native Gateway creates the module. The constructor
-     * should save the {@code broker} parameter.
```

Comment:
In order to publish the broker will still need the moduleAddr param. This should be documented for users who will use the interface rather than the abstract class